### PR TITLE
Use more restrictive IAM policy

### PR DESCRIPTION
**Pull request by vmercierfr**
# Objective

Use more restrictive IAM policy

# Why

Help AWS users to audit and adjust IAM permissions to enable features.

For instance, it will be easier for users who don't collect log information (`collect-logs-size=false`) to remove unnecessary IAM permissions.

Today, we don't specify parameters to collect AWS RDS maintenances, so we can't limit `resource` for `rds:DescribePendingMaintenanceActions` action, but we may support in the future.

# How

- Update IAM policies
- Fix typo in README

# Release plan

- [ ] Merge this PR

### CODE REVIEW

**Comment by vmercierfr** [comment]
I created https://github.com/qonto/prometheus-rds-exporter/issues/96 to fix the issue with `rds:DescribePendingMaintenanceActions` IAM permission

**Comment by vmercierfr** [comment]
#sre

**Comment by vmercierfr** [comment]
For Qonto's review: The policy is successfully deployed in our different AWS accounts.

**Comment by vmercierfr** [comment]
#sre

**Review by dcupif — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -214,13 +252,17 @@ https://raw.githubusercontent.com/qonto/prometheus-rds-exporter/main/configs/aws
 -o /tmp/prometheus-rds-exporter.policy.json
 ```
 
+Terraform uers can take example on Terraform code in `configs/terraform/`.
```
[the remark this answers is not in the document]
Comment (vmercierfr) [reply README.md]: Thanks, I applied the change.

**Pull request merged**